src/app.ts

```diff
@@ -1,6 +1,22 @@
 
 const startApp = async () => {
-    
+    const list = await fetch('https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84')
```

Comment:
Codacy has a fix for the issue: [Replace `··const·list·=·await·fetch('https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84')` with `const·list·=·await·fetch(⏎····"https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84"`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
  const list = await fetch(
```